code/numpy/linalg/linalg.h

```diff
@@ -22,5 +22,6 @@ MP_DECLARE_CONST_FUN_OBJ_1(linalg_cholesky_obj);
 MP_DECLARE_CONST_FUN_OBJ_1(linalg_det_obj);
 MP_DECLARE_CONST_FUN_OBJ_1(linalg_eig_obj);
 MP_DECLARE_CONST_FUN_OBJ_1(linalg_inv_obj);
+MP_DECLARE_CONST_FUN_OBJ_3(linalg_solve_triangular_obj);
```

Comment:
This is not correct: the last argument (`lower`) is actually a keyword in `scipy`.